scripts/zones/Chamber_of_Oracles/mobs/Purson.lua

```diff
@@ -0,0 +1,44 @@
+-----------------------------------
+-- Area: Chamber of Oracles
+--  Mob: Purson
+-- KSNM: The Scarlet King
+-----------------------------------
+---@type TMobEntity
+local entity = {}
+    -- Set Immunities
+entity.onMobInitialize = function(mob)
+    mob:addImmunity(xi.immunity.LIGHT_SLEEP)
+    mob:addImmunity(xi.immunity.DARK_SLEEP)
+    mob:addImmunity(xi.immunity.PETRIFY)
+end
+
+    -- Set Modifiers
+entity.onMobSpawn = function(mob)
+    mob:setMagicCastingEnabled(false) -- Does not cast spells.
```

Comment:
Instead of setting this on lua, an alternative would be to remove its spell list in mob_pools..sql so it has no spells to cast.